msg/CanInterfaceStatus.msg

```diff
@@ -0,0 +1,5 @@
+uint64 timestamp		# time since system start (microseconds)
+
```

Comment:
Should this be per bus?

Reply:
My thought was that is enough to set it up as multi-instance topic, with the instance number mirroring the bus index. But we could probably add a `bus_index` or `bus_id`, like with `device_id` for a lot of other sensors.